src/search/ui/search.html

```diff
@@ -64,6 +64,9 @@
                     <button class="btn btn-default rv-action" ng-click="label()">
                         <i class="fa fa-tag"></i> <span translate-once="search.label"></span>
                     </button>
+                    <button class="btn btn-default rv-action" ng-click="addLanguageIcons()" ng-if="language == 'nl'">
```

Comment:
Why not use a strict comparison?
```suggestion
                    <button class="btn btn-default rv-action" ng-click="addLanguageIcons()" ng-if="language === 'nl'">
```